<a href="https://colab.research.google.com/github/Vitor9988/Trabalho/blob/main/O_vendedor_de_Pirozhki.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# CASE: O vendedor de Pirozhki


Vladimir Sidorov desistiu de seu trabalho convencional na KGB e decidiu empreender em 2010 no ramo alimentício. Com suas economias abriu um foodtruck no bairro do Ipiranga – SP especializado na venda de Pirozhki, onde seu grande diferencial era trabalhar com ingredientes importados dando uma qualidade extraordinária ao produto. Após o crescimento das vendas, a modalidade delivery foi implementada para auxiliar a empresa. Vladimir era extremamente popular no bairro, todos o conheciam, mas em 2019 se mudou para um ponto melhor e não tivemos mais notícia. Os últimos dados compartilhados das vendas de Vlad foram disponibilizados antes de sua partida.




1.   Dado o problema acima, qual o melhor algoritmo para descobrir os dados de 2015?
2.   Qual coeficiente angular?
3.   Considerando uma perspectiva estatística, qual o total de venda em 2020 (Apresentar valores)?
4. Considerando uma perspectiva estatística, qual o total de venda em 2021 (Apresentar valores)?
5. Desenvolva um código em python que calcule estatisticamente as vendas em qualquer ano desejado
6. Considerando sua opinião, qual o total de venda em 2021?
7. Considerando sua opinião, qual o total de venda em 2023?



In [1]:
# 1.
ano_2010=723
ano_2011=814
ano_2012=905
ano_2014=1087
ano_2017=1360
ano_2018=1451
ano_2019=1542

ano_2015=ano_2014+(2015-2014) * 91
print(ano_2015)

1178


In [2]:
#2
ano_2014 = 1087
ano_2012 = 905
# Anos correspondentes
ano_inicio = 2012
ano_fim = 2014
# Cálculo do coeficiente angular
coeficiente_angular = (ano_2014 - ano_2012) / (ano_fim - ano_inicio)
print("Coeficiente Angular:", coeficiente_angular)

Coeficiente Angular: 91.0


In [3]:
# 3
ano_2020=ano_2019+(2020-2019) * 91
print(ano_2020)

1633


In [4]:
# 4
ano_2021=ano_2019+(2021-2019) * 91
print(ano_2021)

1724


In [5]:
# 5
escolha_ano = int(input("Escolha o ano: "))
vendas=ano_2019+(escolha_ano - 2019) * 91
print(vendas)

Escolha o ano: 2040
3453


# 6 e 7
Percebemos que a diferença de vendas entre os anos são de 91 então fizemos um raciocínio logico, fizemos o ano pedido - o ultimo ano que temos registrado e multiplicamos por 91 e somamos com o total do ultimo ano registrado.

2021 = 1724

2023 = 1906



*   Nome:  Gustavo Candile
*   RM: 98852
*   Nome:  Vitor Tavares
*   RM: 99079


